src/biogeophys/SurfaceRadiationMod.F90

```diff
@@ -664,7 +666,9 @@ subroutine SurfaceRadiation(bounds, num_nourbanp, filter_nourbanp, &
                 sabg_soil(p) = sabg(p)
              endif
              ! if no subgrid fluxes, make sure to set both components equal to weighted average
-             if (subgridflag == 0) then 
+!             if (subgridflag == 0) then
+!scs
+             if (subgridflag == 0 .or. lun%itype(l) == istdlak) then
```

Comment:
@swensosc There is one other occurrence of subgridflag, on line 763 of this module. Should that one also check if the landunit type is istdlak?

Reply:
I've pushed this change to the branch